WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -9,6 +9,7 @@
     <% li_classes << "cancelled" if competition.cancelled? %>
     <li class="<%= li_classes.join(' ') %>">
       <span class="date">
+        <%= registration_status_icon(competition) %>
```

Comment:
Could you include this as part of the bigger "if-cascade" below? For example, showing the registration icon doesn't make a lot of sense when the competition is over. So I think you can just replace the last `else` hourglass (the one with the `competitions.index.tooltips.hourglass.starts_in` tooltip) with your new icon. What do you think about that?

Reply:
That makes sense, I have done that in the latest commit